registry/serializers.py

```diff
@@ -1,76 +1,75 @@
-    
     address = AddressSerializer(read_only=True)
+
     class Meta:
         model = Manufacturer
-        fields = ('id', 'full_name','common_name', 'address', 'role')
-        
+        fields = ('id', 'full_name', 'common_name', 'address', 'role')
+
 
 class AuthorizationSerializer(serializers.ModelSerializer):
-    risk_type =  serializers.SerializerMethodField()
-    authorization_type =  serializers.SerializerMethodField()
-    operation_area_type =  serializers.SerializerMethodField()
+    risk_type = serializers.SerializerMethodField()
+    authorization_type = serializers.SerializerMethodField()
+    operation_area_type = serializers.SerializerMethodField()
+
     def get_risk_type(self, obj):
         return obj.get_risk_type_display()
 
     def get_authorization_type(self, obj):
         return obj.get_authorization_type_display()
+
     def get_operation_area_type(self, obj):
         return obj.get_operation_area_type_display()
-    
+
     class Meta:
         model = Authorization
-        fields = ( 'title','risk_type', 'authorization_type', 'operation_area_type', 'end_date')
-          
+        fields = ('title', 'risk_type', 'authorization_type', 'operation_area_type', 'end_date')
 
-class TypeCertificateSerializer(serializers.ModelSerializer):
 
+class TypeCertificateSerializer(serializers.ModelSerializer):
     class Meta:
         model = TypeCertificate
-        fields = ('id', 'type_certificate_id','type_certificate_issuing_country', 'type_certificate_holder','type_certificate_holder_country', )
+        fields = ('id', 'type_certificate_id', 'type_certificate_issuing_country', 'type_certificate_holder',
+                  'type_certificate_holder_country',)
 
-class PersonSerializer(serializers.ModelSerializer):
 
+class PersonSerializer(serializers.ModelSerializer):
     class Meta:
         model = Person
-        fields = ('id', 'first_name','middle_name', 'last_name', 'email','created_at','updated_at')
+        fields = ('id', 'first_name', 'middle_name', 'last_name', 'email', 'created_at', 'updated_at')
 
 
 class TestsSerializer(serializers.ModelSerializer):
-
-    test_type =  serializers.SerializerMethodField()
-    taken_at =  serializers.SerializerMethodField()
+    test_type = serializers.SerializerMethodField()
+    taken_at = serializers.SerializerMethodField()
 
     def get_test_type(self, obj):
         return obj.get_test_type_display()
+
     def get_taken_at(self, obj):
         return obj.get_taken_at_display()
-    class Meta:
-        model = Test
-        fields = ('id', 'test_type','taken_at', 'created_at','updated_at')
 
-class TestsValiditySerializer(serializers.ModelSerializer):
-    
-    tests = TestsSerializer(read_only=True, many=True)
     class Meta:
-        model = TestValidity
-        fields = ('id', 'test' ,'name','created_at','updated_at')
+        model = Test
+        fields = ('id', 'test_type', 'taken_at', 'created_at', 'updated_at')
```

Comment:
This is a error in the original code, the serializer is for pilot test validity, I will fix it in the master